download_posts.py

```diff
@@ -1,16 +1,27 @@
 #!/usr/bin/python3
 
 import json
-import os
-import requests
+import os 
+import requests 
+from sys import exit as sysexit
 import xml.etree.ElementTree as ET
-from auth import cookies, headers
 
+# At time of script import, request range of years. We'll add 1 to the end year automatically.
+try:
+    startYear = int(input("Enter first year you want to export: "))
+except:
+    print("\nError with first year entered. Exiting...")
+    sysexit(1)
 
-YEARS = range(2003, 2015)  # first to (last + 1)
+try:
+    endYear = int(input("Enter last year you want to export: "))
+except:
+    print("\nError with last year entered. Exiting...")
```

Comment:
```suggestion
except Exception as e:
    print(f"\nError with last year entered. Error: {e}. Exiting...")
```

Reply:
Great! The other two files were next on my list. Incoming...